src/utils/numbers/convertEth.js

```diff
@@ -10,27 +10,31 @@ const units = Object.keys(rawUnits).reduce((res, unit) => ({
 }), {})
 
 function convertEth(value: string, from: string, to: string): ?string {
-  const valueBn = toBigNumber(value)
-
-  if (valueBn.isNaN()) {
-    return null
-  }
-
-  const fromValue = from.toLowerCase()
-  if (!units[fromValue]) {
+  try {
+    const valueBn = toBigNumber(value)
+
+    if (valueBn.isNaN()) {
+      return null
+    }
+
+    const fromValue = from.toLowerCase()
+    if (!units[fromValue]) {
+      return null
+    }
+
+    const toValue = to.toLowerCase()
+    if (!units[toValue]) {
+      return null
+    }
+
+    return toBigNumber(valueBn)
+      .mul(units[fromValue])
+      .round(0, BigNumber.ROUND_DOWN)
+      .div(units[toValue])
+      .toString(10)
+  } catch (err) {
     return null
```

Comment:
we really need to validate function's inputs
currently `throw` will hide any bug in this function, even `SyntaxError`